ocs_ci/framework/conf/default_config.yaml

```diff
@@ -51,7 +51,6 @@ DEPLOYMENT:
   # This is described as a WA for minimal configuration 3/3 worker/master
   # nodes. See: https://github.com/openshift/ocs-operator
   ocs_operator_nodes_to_taint: 0
-  ocs_operator_deployment: True
   ssh_key: "~/.ssh/openshift-dev.pub"
   force_deploy_multiple_clusters: False
   allow_lower_instance_requirements: true
```

Comment:
Cannot comment on lower lines, but line: 96 ceph_image, do we still need it there?

Reply:
@clacroix12 Are you planning to address this? we can still use old approvals?

Reply:
@vasukulkarni @petr-balogh I checked for usages of `ceph_image` and it doesn't look like we need it anymore. I removed it locally and am running a local deployment test. If all goes well I will update the PR.

Reply:
Fixed in latest commit